README.md

```diff
@@ -23,7 +23,21 @@ When called the method should print "Would you take this rose, `person`, in exch
 Demonstrate calling the method, passing in "young prince" as the argument.
 
 ```rb
-# Your code goes here...
+require "pry"
+Class Person
+  def initialize (person)
+     @person:person
+  end
+  def offer_rose(person)
+    puts "Would you take this rose, #{@person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
+  end
+end
+
+binding.pry
+# To call the method:
+young_prince=Person.new("offer_rose")
+young_prince.offer_rose
+
 ```
```

Comment:
You don't need to define a class for this question. All you need to define is a method called `offer_rose`. So something like this...

```rb
# method definition
def offer_rose person
  puts "Would you take this rose, #{person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
end

# call method
offer_rose "Adrian"
```